### PR TITLE
feature: add Dockerfiles for PyTorch 1.5.0

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

*Description of changes:*
Adding new Dockerfile for PT1.5.0
Minor changes in mnist as per https://github.com/aws/deep-learning-containers/pull/108.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mmhealey1** [comment]
@TusharKanekiDey will this get merged soon. I would really like to take advantage of v1.5.0 :)

**Comment by TusharKanekiDey** [comment]
@mmhealey1, We are working on it, we will be releasing the PyTorch-1.5 containers soon!

File: buildspec-toolkit.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ env:
   variables:
     FRAMEWORK_VERSION: '1.4.0'
     CPU_INSTANCE_TYPE: 'ml.c4.xlarge'
-    GPU_INSTANCE_TYPE: 'ml.p2.xlarge'
+    GPU_INSTANCE_TYPE: 'ml.p2.8xlarge'
```
Comment (saimidu) [inline buildspec-toolkit.yml:7]: Revert this change as well.

File: docker/1.5.0/py3/Dockerfile.gpu [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    vim \
+    wget \
+    zlib1g-dev \
+ && apt-get remove -y cuda-cufft-dev-10-1 \
+    cuda-curand-dev-10-1 \
+    cuda-cusolver-dev-10-1 \
+    cuda-npp-dev-10-1 \
+    cuda-nvgraph-dev-10-1 \
+    cuda-nvjpeg-dev-10-1 \
+    cuda-nvrtc-dev-10-1
```
Comment (saimidu) [inline docker/1.5.0/py3/Dockerfile.gpu:59]: At the end of this apt-get statement, add the lines `&& apt-get clean \` and `&& rm -rf /var/lib/apt/lists/*`

File: test-toolkit/conftest.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def pytest_addoption(parser):
     parser.addoption('--instance-type')
     parser.addoption('--docker-base-name', default='sagemaker-pytorch-training')
     parser.addoption('--region', default='us-west-2')
-    parser.addoption('--framework-version', default="1.4.0")
+    parser.addoption('--framework-version', default="1.5.0")
```
Comment (saimidu) [inline test-toolkit/conftest.py:55]: Revert this.
